content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -149,12 +139,13 @@ const id = new ObjectId()
 
 ## Known limitations
```

Comment:
OT: Optimally we would present this differently now - these are "just" things done differently in MongoDB, not "limitations" of Prisma with MongoDB really.

Reply:
good point, I changed "limitations" to considerations.